src/sentry/lang/java/utils.py

```diff
@@ -147,3 +147,14 @@ def should_use_symbolicator_for_proguard(project_id: int) -> bool:
         return True
 
     return in_rollout_group(SYMBOLICATOR_PROGUARD_SAMPLE_RATE_OPTION, project_id)
+
+
+def is_jvm_event(data, stacktraces):
```

Comment:
Can we have types for all new functions? It's extremely easy to cause runtime errors since we don't have a unit test for this function (which is ok imho)